docs/howto/obsolete-term.md

```diff
@@ -67,4 +69,10 @@ Consult the wiki documentation for suggestions on standard comments:
  
 1. If the obsoleted term was not replaced by another term in the ontology, but there are existing terms that might be appropriate for annotation, add those term IDs in the 'consider' tag: In the 'Annotations' window, select ```+``` to add an annotation. In the resulting menu, select ```consider``` and enter the ID of the replacement term. 
 
+1. Add any additional axioms needed - this is specific to ontologies and you should consult the conventions of the ontology you are working on. 
```

Comment:
I think the word "annotations" instead of "axioms" is more appropriate here.

Reply:
I agree

Reply:
I will fix